plugins/usbdmx/AsyncUsbReceiver.cpp

```diff
@@ -28,13 +28,16 @@ namespace usbdmx {
 
 AsyncUsbReceiver::AsyncUsbReceiver(ola::usb::LibUsbAdaptor *adaptor,
                                    libusb_device *usb_device,
-                                   PluginAdaptor *plugin_adaptor)
-    : AsyncUsbTransceiverBase(adaptor, usb_device),
+                                   PluginAdaptor *plugin_adaptor,
+                                   unsigned int n_transfers)
```

Comment:
I think num_transfers would probably be more common here based on equivalent code we've got elsewhere.